sklearn/cluster/hierarchical.py

```diff
@@ -644,7 +644,11 @@ class AgglomerativeClustering(BaseEstimator, ClusterMixin):
     pooling_func : callable, default=np.mean
         This combines the values of agglomerated features into a single
```

Comment:
No, it does not. This should just say ignored. When @lesteve pointed to other uses of `.. deprecated` they may be cases where the param is deprecated, but still functional. This one is not.